cypress/support/step-definitions/accessibility/a11y-utils.js

```diff
@@ -0,0 +1,112 @@
+import {
+  commonButtonPreviewRoot,
+  getDataElementByValue,
+} from "../../../locators";
+import { accordionDefaultTitle } from "../../../locators/accordion";
+import {
+  actionPopoverButton,
+  actionPopoverWrapper,
+} from "../../../locators/action-popover";
+import { popoverSettingsIcon } from "../../../locators/popover-container";
+import { visitComponentUrl } from "../../helper";
+import storiesJSON from "../../../../storybook-static/stories.json";
```

Comment:
This `stories.json` file is only generated when running `npx sb extract` so if you try to run any of the Cypress tests locally after just running `npm start` or `npm run build-storybook` they will fail with this error: 

![image](https://user-images.githubusercontent.com/14963680/140925041-54ec33b0-a774-4338-88c1-5a7618bdbf7c.png)

That's for any spec, not just the accessibility ones.

So I think we just need to add some docs somewhere (maybe [here](https://github.com/Sage/carbon/blob/master/docs/testing-styleguide.md#automated-testing) ) which tell the user they need to run `npx sb extract` before `npm run test-cypress`

Reply:
Actually now I've thought about this a bit more, maybe we should just add `npx sb extract` to the `test-cypress` script as it won't work without running it first anyway.

I also think the other Cypress CI in `cypress.yml` might fail without this, unless we added `npx sb extract` in there as well

Reply:
added an info to the `readme.md` file, also added the `config` prop for running `a11y` tests. This issue no more occurs for all regression tests